frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/error-components/types.ts

```diff
@@ -16,9 +16,11 @@ export type Action = {
 
 type Path = {
   id: string;
-  name: string;
-  action: Action;
+  name?: string;
+  nameKey?: string;
+  action?: Action;
```

Comment:
`action` was not optional, is the optional on purpose  ?

Reply:
yes, it was a bug in type definition. See https://github.com/openshift/console/pull/7378/files#diff-162808b7604666ef2098e1a226ed105228ba4b9858e4aaa741e2d4d487c6fd15R31 where Error path does not contain `action`